src/main/java/org/csanchez/jenkins/plugins/kubernetes/ContainerTemplate.java

```diff
@@ -176,6 +176,10 @@ public boolean isPrivileged() {
     }
 
     @DataBoundSetter
+    public void setRunAsUser(String runAsUser) {
+        this.runAsUser = PodTemplateUtils.parseLong(runAsUser);
+    }
+
     public void setRunAsUser(Long runAsUser) {
```

Comment:
Then delete the original setter please. And ensure the types of getter and setter are identical.